cypress/e2e/init.spec.ts

```diff
@@ -0,0 +1,10 @@
+describe('Cypress', () => {
+  it('is working', () => {
+    expect(true).to.equal(true);
+  });
+
+  it('visits the app', () => {
+    cy.visit('/');
```

Comment:
Let's add an extra line after this one to separate the setup and assertion part.

Reply:
Added in [adc7753](https://github.com/rosle/react-nimble-survey/pull/9/commits/adc77533885dae70bb6ecf05fefbe95041fceff0) ✨